src/lib/utils.ts

```diff
@@ -1,4 +1,4 @@
-import { pseudoRandomBytes } from 'crypto';
```

Comment:
`pseudoRandomBytes` has been deprecated for a long time since it was made to match `randomBytes` and is not available when importing from ESM